package.json

```
@@ -1,7 +1,7 @@
 {
   "scripts": {
     "fmt": "biome format --write . && biome check standalone_app vscode tslib --apply",
-    "lint": "npm run lint:eslint && npm run lint:biome",
+    "lint": "npm run lint:eslint && npm run lint:biome && cd optuna_dashboard && npm run lint",
     "lint:eslint": "eslint . --ext .ts,.tsx --max-warnings 0",
     "lint:biome": "biome format . && biome ci standalone_app vscode tslib"
```

nit: How about calling `npm run type-check` directrly?

```suggestion
    "lint": "npm run lint:eslint && npm run lint:biome && npm run lint:tsc",
    "lint:eslint": "eslint . --ext .ts,.tsx --max-warnings 0",
    "lint:biome": "biome format . && biome ci standalone_app vscode tslib"
    "lint:tsc": "cd optuna_dashboard && npm run type-check"
```